server/lib/settings.ts

```diff
@@ -16,6 +16,7 @@ export interface PlexSettings {
   ip: string;
   port: number;
   useSsl?: boolean;
+  presetServer?: boolean;
```

Comment:
Is this setting actually used for anything?

Reply:
Yes, it is, if the server was configured manually, the form is read/write, whereas if it was configured automatically the form becomes read only, unless you change the preset to manual - storing it server side stops people who manually set the server to the plex direct URL from being unable to modify it, however, it only needs to be updated on form submit.

Reply:
Is there any reason we actually must show the input boxes if they have as preset server selected? We can always just hide them in the case thats its preset and set the formik values manually with `setFieldValue` and then we don't really even need this `presetServer` config option.

Reply:
When the component renders we can determine again if its one of the preset servers, no? So in that case we just don't render the inputs again.

Reply:
I tried multiple ways to deal with this - because the available presets is loaded after the page loads, the most sensible way is to just default to manual on page load - server side variable has been removed.